admiral/cmd/admiral/cmd/root.go

```diff
@@ -71,6 +71,14 @@ func GetRootCmd(args []string) *cobra.Command {
 		"Prefix to use when creating SAN for Service Entries")
 	rootCmd.PersistentFlags().StringVar(&params.SecretResolver, "secret_resolver", "",
 		"Type of resolver to use to fetch kubeconfig for monitored clusters")
+	rootCmd.PersistentFlags().StringVar(&params.LabelSet.DeploymentLabel, "deployment_label", "istio-injected",
```

Comment:
This should be `sidecar.istio.io/inject: true` annotation (which will also fix the bug)?